### PR TITLE
Tag Propagation

### PR DESCRIPTION
**Pull request by kcooperstein**

### CODE REVIEW

File: opencensus/tags/tag_value.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+        :type value: str
+        :param value: the value to be checked
+
+        :rtype: bool
+        :returns: True if valid, if not, False.
+
+        """
+        if len(value) <= 255:
+            if (all(ord(char) < 126 for char in value) and
```
Comment (zeako) [inline opencensus/tags/tag_value.py]: could rewrite as
```py
return (all(32 < ord(char) < 126 for char in value)
```

File: docs/trace/usage.rst [hunk 2]
```diff
@@ -182,8 +182,8 @@ For Django, you can configure the blacklist in the ``OPENCENSUS_PARAMS`` in ``se
 Framework Integration
 ---------------------
 
-Opencensus supports integration with popular web frameworks including
-Django, Flask, and Webapp2. When the application receives a HTTP request,
+Census supports integration with popular web frameworks including
```
Comment (SergeyKanzhelev) [inline docs/trace/usage.rst]: Keep OpenCensus here?

**Comment by SergeyKanzhelev** [comment]
Any chance you can split this PR into separate smaller ones? Too much to review in a single shot

**Comment by songy23** [comment]
@kcooperstein looks like you have conflicts between this branch and master, can you rebase against upstream/master? (assume upstream is https://github.com/census-instrumentation/opencensus-python.git)

**Comment by liyanhui1228** [comment]
Thanks Katy for implementing this! I'm taking this PR over and fixed the python2 test failures as well as lint and coverage warnings. Ignoring the CLA warning as it seems not support multiple authors contributing to one PR.

**Pull request merged**